src/documentation/config/config.md

```diff
@@ -38,6 +38,12 @@ Each configuration option can be:
 
 		columnWidth: function() { return this.rangeBand/3 * 2; }
 
+	__Note__ Function parameters will be invoked just before each render happens or in the within the render cycle with the corresponding data point in the following cases:
```

Comment:
do you mean _except_ the following cases, or am i mis-understanding what this is?

Reply:
I changed the text a bit to make it clearer